tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/AdoNetClientInstrumentMethodsAttribute.cs

```diff
@@ -288,7 +294,8 @@ internal class CommandExecuteScalarWithBehaviorAttribute : Attribute
         [AdoNetTargetSignature(
             MethodName = AdoNetConstants.MethodNames.Read,
             ReturnTypeName = ClrNames.Bool,
-            CallTargetType = typeof(ReaderReadIntegration))]
+            CallTargetType = typeof(ReaderReadIntegration),
+            InstrumentationCategory = InstrumentationCategory.Iast)]
```

Comment:
I know these are only used for `Iast` currently, but I think _defaulting_ to Iast here is confusing when everything else is for `Tracing` by default.

I think we should rename these attributes to make it clear, e.g.

`ReaderReadAttribute` -> `ReaderReadIastAttribute` or `IastReaderReadAttribute`

As we'll need to create new attributes if we want to target tracing etc.

Reply:
Glad you saw it. It's a minstake. Default should be Tracer. Although true default is well set in the Attribute definition. But good catch.

Reply:
Ok, I see your point.